index.css

```diff
@@ -41,12 +41,13 @@ p {
     margin: 0;
 }
 
-#wrapper {
+#sub-menu-wrapper {
     border: 2px solid red;
-    height: 463px;
+    height: 94%;
 }
 
-#internet-contents-wrapper-element {
+#internet-contents-wrapper-element,
+#applications-wrapper-element {
```

Comment:
If 2 elements have similar CSS, this means that you need class

Reply:
Solved